src/test-utils/OBZFixture.ts

```diff
@@ -0,0 +1,20 @@
+export class OBZFixture {
+
+  constructor() {
+  }
+
+  static load = (fixtureName) => {
+    // pull fixture from window.__obz__ array
+    const fixtures = window['__obz__'];
+    const noManifest = fixtures[fixtureName];
```

Comment:
if this is for more than just noManifest can this variable please be renamed to something more generic?